examples/build_report_template.html

```diff
@@ -69,29 +69,48 @@ <h2>Details for <a href="{{ base_url + 'build/' + build.version + '/' }}">build
             </div>
             <div id="testrun-list">
                 <h2>LAVA test results</h2>
-                {% for env_name, env_results in environments.items() %}
-                <h3>{{ env_name }}</h3>
+                {% for envs_tr_tuple, env_results in envs_tr_combo.items() %}
+                <h3>{{ envs_tr_tuple[0] }}</h3>
                 <table class="table" border="1">
+                    <tr>
```

Comment:
If at all, this should be outside of the table. Also giving just the test run ID seems a bad idea.